openupgrade_scripts/scripts/hr/17.0.1.1/post-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Viindoo Technology Joint Stock Company (Viindoo)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+_deleted_xml_records = [
+    "hr.dep_sales",
```

Comment:
In fact, don't delete this record with delete_records_safely_by_xml_id, because the record was moved to demo data, but note deleted.

For `hr.dep_sales`, I think you should delete only the xmlid (not the record) with:
```python
env["ir.model.data"].search(
                [("module", "=", "hr"), ("name", "=", "dep_sales")], limit=1
            ).unlink()
```

Reply:
I think it's correct to delete safely. Demo data are not in production databases, so at the end, if the department is not used, it can be deleted. If not, the method will do the rest.